.github/scripts/update_disabled_tests.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+"""
+Query for the DISABLED test issues.
+
+"""
+
+import json
+from functools import lru_cache
+from typing import Any, Dict
+from urllib.request import urlopen, Request
+
+# Modified from https://github.com/pytorch/pytorch/blob/b00206d4737d1f1e7a442c9f8a1cadccd272a386/torch/hub.py#L129
+def _read_url(url: Any) -> Any:
+    with urlopen(url) as r:
+        return r.headers, r.read().decode(r.headers.get_content_charset("utf-8"))
+
+
+def request_for_labels(url: str) -> Any:
+    headers = {"Accept": "application/vnd.github.v3+json"}
+    return _read_url(Request(url, headers=headers))
+
+
+def get_last_page(header: Any) -> int:
+    # Link info looks like:
+    # Link: <https://api.github.com/search/issues?q=is%3Aissue+is%3Aopen+repo%3Apytorch%2Fpytorch+in%3Atitle+
```

Comment:
I think this comment can be removed since line 45 of code has this information

Reply:
Oh this was to explain the parsing code later on in the function